userguide/dailyuse/anbox.rst

```diff
@@ -26,7 +26,7 @@ How to install
 .. warning::
     Installing Anbox is only recommended for experienced users.
 
-- Make sure your supported device runs on 16.04 (Anbox doesn't work on 15.04)
+- Make sure your supported device runs on 16.04 (Anbox doesn't work on 15.04); on `dev` channel be sure to have a backup of the device.
```

Comment:
I'm still not convinced that there is anything specific about the channel. If we want to warn about backups, I think we should do it without the dev disclaimer. 

Btw, note that there is a recently merged warning about re-flashing anbox boot img

Reply:
>Btw, note that there is a recently merged warning about re-flashing anbox boot img
what do you mean?

Reply:
https://github.com/ubports/docs.ubports.com/pull/262